saleor/graphql/meta/mutations/utils.py

```diff
@@ -4,19 +4,22 @@
 from ....core.error_codes import MetadataErrorCode
 
 
-def save_instance(instance, metadata_field: str):
-    fields = [metadata_field]
-
+def save_instance(instance, metadata_fields: list, commit: bool = False):
```

Comment:
Do we need this `commit` parameter? Are we using `False` somewhere?

Reply:
thought it might be helpful in future use to avoid multiple saves, but in this implementation it is not used - do you think it should be removed? what is the approach - better not to add not used features?

Reply:
This function is responsible only for saving mutation instances. This function does nothing without committing. We use this pattern with `commit` mostly in functions inside Models.

Reply:
ok so I will remove commit flag if it is redundant here